src/OrchardCore.Modules/Orchard.Admin/Orchard.Admin.csproj

```diff
@@ -18,4 +18,8 @@
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="1.1.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <DotNetCliToolReference Include="Microsoft.DotNet.Watcher.Tools" Version="1.0.0" />
```

Comment:
So, the project you want to watch needs to reference the tool, and you need to execute the tool from the directory that contains the project.

So, if we only reference the tool in the web app and start it from there, there are too many project references and it is very slow.

So, that's why we reference the tool in each module and, through a batch file, we start the tool from each directory of each module we want to watch.